package.json

```diff
@@ -8,7 +8,6 @@
     "build": "concurrently -c \"magenta,blue\" \"npm:theme.bundle.prod\" \"npm:print.bundle\"",
     "format": "concurrently -c \"magenta,blue,green\" \"npm:format.tokens\" \"npm:format.assets\" \"npm:format.svg\"",
     "format.assets": "eslint assets --fix && prettier --write assets",
-    "format.svg": "svgo -f ./assets/images/svg/icons --config=./svgo.config.js",
```

Comment:
@komejo Why take this part out? It's tidying up the svg icons, removing height/width and things like comments/styles, but even if that's not desired, the `npm run format` command still references it.

Reply:
It's important for a number of use cases to keep the SVG height & width, as well as other custom properties & styles that this strips out. I'll update the `npm run format` command.